fixtures/shehds/led-flat-par-7x18w-rgbwa-uv-light.json

```diff
@@ -0,0 +1,164 @@
+    "authors": ["JMS", "le 11/10/2021", "jms27000"],
+    "createDate": "2022-10-21",
+    "lastModifyDate": "2022-10-21",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2021-10-16",
+      "comment": "created by Q Light Controller Plus (version 4.12.4)"
+    }
+  },
+  "links": {
+    "productPage": [
+      "https://www.shehds.com/products/led-flat-par-7x12w-7x18w-rgbwauv-lighting"
+    ],
+    "manual": [
+      "https://www.shehds.com/u_file/2005/file/b01e72a1b3.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [176, 178, 11.3],
+    "DMXconnector": "3-pin"
+    "bulb": {
+      "type": "LED"
+    },
+    "power": 126,
+    "weight": 0.9,
+  },
+  "availableChannels": {
+    "Master dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "Violet": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "speedStart": "0Hz",
+        "speedEnd": "10Hz",
+        "helpWanted": "At which DMX values is the strobe effect disabled?"
+      },
+    },
+    "Function": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 50],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [51, 100],
+          "type": "Effect",
+          "effectName": "Auto Mode 1",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [101, 150],
+          "type": "Effect",
+          "effectName": "Auto Mode 2",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [151, 200],
+          "type": "Effect",
+          "effectName": "Auto Mode 3",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [201, 255],
+          "type": "Effect",
+          "soundControlled": true,
+          "effectName": "Sound Mode",
+          "helpWanted": "At which DMX value does the effect start?"
+        }
+      ]
+    },
+    "Function Speed": {
```

Comment:
This affects the previous channel, but I wasn't sure how to reference it? I took a look at the schema but couldn't figure it out.

Reply:
Use the `EffectSpeed` capability type, then it's implicit that the effect is controlled. There is no more explicit way to link Effect+EffectSpeed channels in the schema yet.